src/cascadia/TerminalSettings/terminalsettings.h

```diff
@@ -88,6 +88,9 @@ namespace winrt::Microsoft::Terminal::Settings::implementation
         hstring StartingTitle();
         void StartingTitle(hstring const& value);
 
+        bool SuppressApplicationTitle();
+        void SuppressApplicationTitle(bool value);

```

Comment:
Do these need to be here at all? Doesn't look like they're being used by `TerminalPage` at all.

Typically this settings class is only used for `ICoreSettings` or `IControlSettings`, and I don't think `SuppressApplicationTitle` is either of those, it's more of an _app_ setting